app/controllers/weets.js

```diff
@@ -0,0 +1,24 @@
+const logger = require('../logger');
+const { postWeetSerializer } = require('../serializers/weets');
+const { getQuote, createWeet } = require('../services/weets');
+const { mapToSerializer } = require('../utils/objects');
+
+const loggerPath = 'controller:weet';
+
+exports.postWeet = async (req, res, next) => {
+  try {
+    const { id: userId } = req.user;
+    logger.info(`${loggerPath}:postWeet: starting postWeet method with userId - ${userId}}`);
+    const { joke: content } = await getQuote();
+    logger.info(
+      `${loggerPath}:postWeet: continue postWeet method with content - ${JSON.stringify(content)}}`
+    );
+
+    const weetData = { content, userId };
+    const weet = await createWeet(weetData);
```

Comment:
Before creating the weet, you could validate that it is a maximum of 140 characters.